src/assets/TokenDetectionController.ts

```diff
@@ -157,19 +207,12 @@ export class TokenDetectionController extends BaseController<
     }, this.config.interval);
   }
 
-  /**
-   * Checks whether network is mainnet or not.
-   *
-   * @returns Whether current network is mainnet.
-   */
-  isMainnet = (): boolean => this.config.networkType === MAINNET;
```

Comment:
This was the only place `this.config.networkType` was used. It can be removed now.

Reply:
Good catch. Removed code